formula.yaml

```diff
@@ -11,606 +11,16 @@ buildSettings:
-    artifactId: token-macro
-    source:
-      version: "2.13"
-  - groupId: org.jenkins-ci.plugins
-    artifactId: trilead-api
-    source:
-      version: 1.0.13
-  - groupId: org.jenkins-ci.plugins
-    artifactId: variant
-    source:
-      version: "1.4"
-  - groupId: org.jenkins-ci.plugins
-    artifactId: windows-slaves
-    source:
-      version: "1.7"
-  - groupId: org.jenkins-ci.plugins
-    artifactId: ws-cleanup
-    source:
-      version: "0.38"
-  - groupId: org.jenkins-ci.plugins.pipeline-stage-view
-    artifactId: pipeline-stage-view
-    source:
-      version: "2.18"
-  - groupId: org.jenkins-ci.plugins.workflow
-    artifactId: workflow-aggregator
-    source:
-      version: "2.6"
-  - groupId: org.jenkins-ci.plugins.workflow
-    artifactId: workflow-api
-    source:
-      version: 1108.v57edf648f5d4
-  - groupId: org.jenkins-ci.plugins.workflow
-    artifactId: workflow-basic-steps
-    source:
-      version: "2.22"
-  - groupId: org.jenkins-ci.plugins.workflow
-    artifactId: workflow-cps
-    source:
-      version: 2648.va9433432b33c
-  - groupId: org.jenkins-ci.plugins.workflow
-    artifactId: workflow-cps-global-lib
-    source:
-      version: 2.19
-  - groupId: org.jenkins-ci.plugins.workflow
-    artifactId: workflow-durable-task-step
-    source:
-      version: "2.36"
-  - groupId: org.jenkins-ci.plugins.workflow
-    artifactId: workflow-job
-    source:
-      version: 2.41
-  - groupId: org.jenkins-ci.plugins.workflow
-    artifactId: workflow-scm-step
-    source:
-      version: 2.13
-  - groupId: org.jenkins-ci.plugins.workflow
-    artifactId: workflow-step-api
-    source:
-      version: 615.vb09dac339255
-  - groupId: org.jenkins-ci.plugins.workflow
-    artifactId: workflow-support
-    source:
-      version: "3.8"
-  - groupId: org.jenkins-ci.ui
-    artifactId: jquery-detached
-    source:
-      version: 1.2.1
-  - groupId: org.jenkinsci.plugins
-    artifactId: pipeline-model-definition
-    source:
-      version: 1.9.3
-  - groupId: org.jenkinsci.plugins
-    artifactId: pipeline-model-extensions
-    source:
-      version: 1.9.3
-  - groupId: org.jenkins-ci.plugins.workflow
-    artifactId: workflow-multibranch
-    source:
-      version: 2.24
-  - groupId: org.jenkins-ci.plugins
-    artifactId: label-linked-jobs
-    source:
-      version: "6.0.1"
-  - groupId: org.jenkins-ci.plugins
-    artifactId: prometheus
-    source:
-      version: "2.0.10"
-  - groupId: org.jenkins-ci.plugins
-    artifactId: cloudbees-disk-usage-simple
-    source:
-      version: "0.10"
+  pom: pom.xml
+  pomIgnoreRoot: true
+  pomIncludeWar: true
 systemProperties:
-  { hudson.security.csrf.DefaultCrumbIssuer.EXCLUDE_SESSION_ID: "true" }
+  hudson.security.csrf.DefaultCrumbIssuer.EXCLUDE_SESSION_ID: "true"
+  file.encoding: UTF-8
+  sun.jnu.encoding: UTF-8 
+  jenkins.install.runSetupWizard: "false"
+  org.apache.commons.jelly.tags.fmt.timeZone: Asia/Shanghai
+  user.timezone: Asia/Shanghai
```

Comment:
These settings might not be friendly enough for other users who are not in China.

It could be an option of [the Helm chart](https://github.com/kubesphere-sigs/ks-devops-helm-chart).